doc/integration/integration-prometheus-grafana.md

```diff
@@ -0,0 +1,176 @@
++ StarRocks operator and helm chart: v1.7.1+
+
+## 2. Deploy Prometheus Service
+
+> SKIP this step if the Prometheus service is already available in the Kubernetes environment.
+
+Follow the instructions from [Prometheus Helm Chart](https://artifacthub.io/packages/helm/prometheus-community/prometheus) to deploy the Prometheus service into the Kubernetes environment.
+
+```shell
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+helm install prometheus prometheus-community/prometheus
+```
+
+## 3. Deploy Grafana Service
+
+> SKIP this step if the Grafana service is already available in the Kubernetes environment.
+
+Follow the instructions from [Grafana Helm Chart](https://artifacthub.io/packages/helm/grafana/grafana) to deploy the Grafana service into the Kubernetes environment.
+
+Need override the default value of the grafana service, change the Grafana service type from default `ClusterIP` to `LoadBalancer` in order to access the Grafana service from the outside Kubernetes network.
+
+An example `grafana-values.yaml` may look like as the following snippet.
+```yaml
+service:
+  enabled: true
+  type: "LoadBalancer"
+```
+
+Install the Grafana service with the following commands.
+```shell
+helm repo add grafana https://grafana.github.io/helm-charts
+helm repo update
+helm install grafana -f grafana-values.yaml grafana/grafana
+```
+
+## 4. Login And Configure Grafana Service
+
+> SKIP this section if the prometheus data source is already configured in Grafana.
+
+### 4.1 Get Grafana Service External Address
+
+Run the following command to get the external ip address
+```shell
+kubectl get svc grafana
+```
+A possible result may look like the follows
+```text
+$ kubectl get svc grafana
+NAME      TYPE           CLUSTER-IP    EXTERNAL-IP     PORT(S)        AGE
+grafana   LoadBalancer   10.40.6.237   35.239.53.124   80:31362/TCP   4m25s
+```
+### 4.2 Get Grafana Admin Password
+
+Run the following command to get the login password for admin user.
+```shell
+kubectl get secrets grafana -o yaml | grep admin-password | awk '{print $2}' | base64 -d
+```
+### 4.3 Login Grafana and Configure Prometheus Data Source
+
+Access Grafana web GUI with `http://<external_ip>/`, login with `admin:<password>`.
+
+From the navigation path, `"Administration" -> "Configuration" -> "Add Data Source"`, set the HTTP.URL to `http://prometheus-server.default`. Change `default` to the correct namespace if the prometheus service is not deployed in the default namespace. Refer to Grafana doc [here](https://grafana.com/docs/grafana/latest/datasources/prometheus/configure-prometheus-data-source/) for detailed instructions.
+
+## 5. Deploy StarRocks Cluster
+
+Follow the instructions from [StarRocks Helm Chart](https://artifacthub.io/packages/helm/kube-starrocks/kube-starrocks) with some customized values.
+
+Following is an example of the content of the `sr-values.yaml`.
+
+```yaml
+starrocksFESpec:
```

Comment:
should be indented and placed below `starrocks`

Reply:
good catch

Reply:
add examples for both 1.7.1 and 1.8.0